chain/jsonrpc-primitives/src/types/split_storage.rs

```diff
@@ -39,3 +41,16 @@ impl From<RpcSplitStorageInfoError> for crate::errors::RpcError {
         Self::new_internal_or_handler_error(error_data, error_data_value)
     }
 }
+
+impl RpcSplitStorageInfoError {
+    // Implementing From<RpcSplitStorageInfoError> for RpcStatusError causes cargo to spit out hundreds
+    // of lines of compilation errors. I don't want to spend time debugging this, so let's use this function instead.
+    // It's good enough.
```

Comment:
lol
if you want we can try together, for the sake of learing rust but as you said it's just fine as is

Reply:
Eh I'm fine with leaving it as is. The errors looked pretty intimidating, and there is no point in spending mental energy on trying to figure out what it is complaining about. "Focus on what matters", right? :P

Reply:
I'd even argue that `into_rpc_status_error` is more readable than just `into` :P